manim/mobject/svg/tex_mobject.py

```diff
@@ -313,8 +313,7 @@ class Title(Tex):
     }
 
     def __init__(self, *text_parts, **kwargs):
-        TextMobject.__init__(self, *text_parts, **kwargs)
-        self.underline_width = config["frame_width"] - 2
```

Comment:
Isnt this necessary as well?

Reply:
I was wondering the same thing. I thought you had removed it and so that's why I removed it from my branch too :thinking: 

I'll put it back again :smile: